### ISSUE TITLE
[BUG] Incorrect value on \G notation

### ISSUE THREAD

**Issue by mercurykd** [cweijan/vscode-mysql#911]
- Operating system: win11 x64
- Database type and version: 10.6.4-MariaDB-1:10.6.4+maria~bionic-log

When you select everything through *, the values ​​are displayed incorrectly.
SQL for test:
```sql
select * from (select 1 id, 'first' title) t1
join (select 1 id, 'two' title) t2\G
```
<img src="https://github.com/cweijan/vscode-database-client/assets/30900414/dc8c1b09-658f-4aa5-be4b-33a15b8231e5" width="300">

**Comment by cweijan** [comment]
Thanks for the feedback, I will fix it in the next version.

**Comment by cweijan** [comment]
Fixed in version 6.7.1.
